### PR TITLE
add orca optimizesrs

### PR DESCRIPTION
**Pull request by yangw1234**

### CODE REVIEW

File: pyzoo/zoo/orca/learn/optimizers.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+                 learningrates=None,
+                 weightdecays=None):
+        from bigdl.optim.optimizer import SGD as BSGD
+        self.optimizer = BSGD(learningrate,
+                              learningrate_decay,
+                              weightdecay,
+                              momentum,
+                              dampening,
+                              nesterov,
+                              leaningrate_schedule.get_scheduler(),
```
Comment (jason-dai) [inline pyzoo/zoo/orca/learn/optimizers.py:62]: check the type of `leaningrate_schedule`

File: pyzoo/test/zoo/orca/learn/spark/test_estimator_keras_for_spark.py [hunk 2]
```diff
@@ -475,8 +475,8 @@ def test_estimator_keras_with_bigdl_optim_method(self):
                                                       np.random.randint(0, 50, size=(100, 1)),
                                                       np.ones(shape=(100,), dtype=np.int32)))
         dataset = dataset.map(lambda user, item, label: [(user, item), label])
-        from bigdl.optim.optimizer import SGD
-        from bigdl.optim.optimizer import Plateau
+        from zoo.orca.learn.optimizers import SGD
+        from zoo.orca.learn.schedule import Plateau
```
Comment (jason-dai) [inline pyzoo/test/zoo/orca/learn/spark/test_estimator_keras_for_spark.py:479]: `orca.learn.optimizers.schedules`

File: pyzoo/test/zoo/orca/learn/spark/test_estimator_for_spark.py [hunk 3]
```diff
@@ -20,10 +20,10 @@
 
 import numpy as np
 import tensorflow as tf
-from bigdl.optim.optimizer import SeveralIteration
 from pyspark.sql.context import SQLContext
 
 import zoo.orca.data.pandas
+from bigdl.optim.optimizer import SeveralIteration
```
Comment (jason-dai) [inline pyzoo/test/zoo/orca/learn/spark/test_estimator_for_spark.py:26]: use orca.learn.Trigger

File: pyzoo/test/zoo/orca/learn/spark/test_estimator_keras_for_spark.py [hunk 4]
```diff
@@ -20,8 +20,8 @@
 import tensorflow as tf
 
 from bigdl.optim.optimizer import SeveralIteration
```
Comment (jason-dai) [inline pyzoo/test/zoo/orca/learn/spark/test_estimator_keras_for_spark.py:22]: use orca.learn.Trigger

**Comment by yangw1234** [comment]
http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/4556/ jenkins passed

**Pull request merged**